build/edgesite/edgesite-server.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: Pod
```

Comment:
ditto

Reply:
now we only use one pod, no need to use deployment

Reply:
Please use deployment, thanks.

Reply:
done

Reply:
done